framework/xds_k8s_testcase.py

```diff
@@ -386,6 +388,40 @@ def removeServerBackends(self, *, server_runner=None):
         # Remove backends from the Backend Service
         self.td.backend_service_remove_neg_backends(neg_name, neg_zones)
 
+    def assertHealthyEndpointsCount(
+        self,
+        test_client,
+        expected_len,
+        msg=None,
```

Comment:
Is there a need for msg? see assertDrainingEndpointsCount. It's better to provide the default message, same as here: https://github.com/grpc/psm-interop/blob/8b8016ea3ae2bfb8cbade7d7521fbc9d2c2f0fee/framework/xds_k8s_testcase.py#L677-L681